gap_analysis_price/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_price}
+==============
+
+have price/cost estimation for business requirements.
```

Comment:
This module extends the gap analysis module set to allow
price estimation for the analysis, based on the time estimation and 
cost structure (resource, task type and price).